pubspec.yaml

```diff
@@ -17,6 +17,7 @@ dependencies:
   collection: ^1.18.0
   cupertino_icons: ^1.0.8
   dio: ^5.7.0
+  equatable: ^2.0.5
```

Comment:
We don't need `equatable`, we already have `freezed` for the function this is trying to achieve in:
 
- lib/search/cubit/search_state.dart
- lib/search/models/search_result.dart

Samples: 
- lib/features/auth/cubit/social_auth_sign_in_state.dart
- lib/common/data/models/auth.dart

Reply:
Okay, this has been replaced